ggml-quants.c

```diff
@@ -3677,7 +3677,7 @@ void ggml_vec_dot_q2_K_q8_K(const int n, float * restrict s, const void * restri
 
         const uint8x16_t mins = vshrq_n_u8(mins_and_scales, 4);
         const ggml_int16x8x2_t q8sums = ggml_vld1q_s16_x2(y[i].bsums);
-        const ggml_int16x8x2_t mins16 = {vreinterpretq_s16_u16(vmovl_u8(vget_low_u8(mins))), vreinterpretq_s16_u16(vmovl_u8(vget_high_u8(mins)))};
+        const ggml_int16x8x2_t mins16 = {{vreinterpretq_s16_u16(vmovl_u8(vget_low_u8(mins))), vreinterpretq_s16_u16(vmovl_u8(vget_high_u8(mins)))}};
```

Comment:
What does this do?

Reply:
I guess it addresses this warning on 32-bit ARM builds:

<img width="1155" alt="Screenshot 2023-12-22 at 12 56 28" src="https://github.com/ggerganov/llama.cpp/assets/1991296/cdcd031c-40ae-4001-9556-08a99221f92c">